docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,119 @@
+
+### When does a billing cycle start for a plan?
+
+For the Free plan, the billing cycle starts on the first day of the calendar month.
+
+For all paid plans, the billing cycle starts from the subscription date of that plan.
+
+### How do I update my billing information or add a tax ID?
+
+To update your Organization account's billing information or add a tax ID, make sure that you have either an [**Owner** or **Admin** role](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role. After confirming your role:
+
+- Go to your [**Account or Organization settings** > **Billing**](https://expo.dev/settings/billing), and click on **Update billing information**. This will take you to Stripe's portal.
+- On Stripe's portal, under **Billing information**, click on **Update information** to update billing-related information such as billing name, email, address, and tax ID.
+
+See [Manage billing information](/billing/manage/#manage-billing-information) for more details.
+
+### Can I update the billing info on my last invoice?
+
+No. Updating billing information will only be reflected on the next invoice.
+
+### Can you email me the receipts?
+
+No. Your account's Owner or Admin can download them. See [Download a receipt](/billing/invoices-and-receipts/#download-and-view-a-receipt) for more information.
+
+### How can I reduce the amount of EAS Build usage by using EAS Update?
+
+Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. For native code changes, you can also automate the build process using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime).
+
+For more information, see [How to optimize build usage](/billing/usage-based-pricing/#how-to-optimize-build-usage).
+
+### How do I estimate my next bill?
+
+To estimate your next bill, go to [**Billing**](https://expo.dev/settings/billing). Under **Usage this month**, you will find a summary of your EAS Build usage based on the [resource class](/build/eas-json/#selecting-resource-class), EAS Update usage based on monthly updated users and global edge bandwidth, and the amount spent for both.
+
+See [How usage-based pricing works](/billing/usage-based-pricing/#how-usage-based-pricing-works) for more information.
+
+### What is a monthly updated user?
+
+A monthly updated user is a unique user who downloads an update via EAS Update in a single month, regardless of how many updates this user downloads in a given billing month/period. See [How are "monthly updated users" counted for a billing cycle](/eas-update/faq/#how-are-monthly-updated-users-counted-for-a-billing-cycle) for more information.
```

Comment:
```suggestion
A monthly updated user is a unique user who downloads an update via EAS Update in a single month, regardless of how many updates this user downloads in a given billing period. See [How are "monthly updated users" counted for a billing period](/eas-update/faq/#how-are-monthly-updated-users-counted-for-a-billing-cycle) for more information.
```
I originally thought about switching "cycle" to period when I saw the "month/period" here, made me think about making that consistent, but then I saw that "cycle" was used more in the other docs. So, don't want to comment it every time, but want to make a general comment about making those consistent. FWIW, I think I most frequently refer to it as a "billing period" and AFAIK, the terms are interchangeable. Though, now I'm reading [this forum post](https://forums.att.com/conversations/uverse-tv-account/billing-period-billing-cycle/5defc6e9bad5f2f6062c6a7f) and some are saying that the billing period is the segment of time you're billed FOR, and the billing cycle is the length of time between when you RECEIVE the bills. If that's the case, a) our period and cycle are almost entirely the same, but b) in the context we generally use it, "period" is a little more accurate.

Reply:
That's a thorough investigation! Thanks for doing that! I'll update all instances to use "period" instead of cycle. I just found in our dashboard > Usage page, we also use the term "period" as in "Current billing period" instead of cycle. Another good point for us to be consistent.